src/webview/LeetCodeWebview.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) jdneo. All rights reserved.
+// Licensed under the MIT license.
+
+import { ConfigurationChangeEvent, Disposable, ExtensionContext, ViewColumn, WebviewPanel, window, workspace } from "vscode";
+import { markdownEngine } from "./markdownEngine";
+
+export abstract class LeetCodeWebview implements Disposable {
+
+    protected panel: WebviewPanel | undefined;
+    private context: ExtensionContext;
+    private listener: Disposable;
+
+    public initialize(context: ExtensionContext): void {
+        const { onDidChangeConfiguration } = this.getWebviewOption();
```

Comment:
Why not make `onDidChangeConfiguration` as a class member?

Reply:
I wanted to make the way to implement two event callback consistent. After some investigation on js overriden mechanism, now I made sure all two methods could be set as class member.